src/api/app/assets/stylesheets/webui/project-package-show-grid.scss

```diff
@@ -6,7 +6,7 @@
   grid-area: build-results;
 }
 
-.comments {
+#comments-list {
```

Comment:
not possible to keep it as `comments` ?

Reply:
At this point I need an anchor to let the notification point to the beginning of the comments box.
As there is another `#comments` in the same view (see `card-body`), I needed to use another ID to go to the top of the card (including card header).